# A person has some body parts. That is why we have created
# an array of body parts.
Bodypart[] bodyParts 
